woostarter/patterns/about-product-features-2-columns.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Title: Product features, 2 columns
+ * Slug: woostarter/product-features-2-columns
+ * Categories: text, about, media
+ * Keywords: about, text, icons
+ * Description: A section with a heading, paragraph, product features with icons and product photo.
+ *
+
+?>
+<!-- wp:group {"metadata":{"categories":["intro"],"name":"Heading, image and product features","patternName":"a8c/intro-two-column-with-image-and-content"},"align":"full","className":"alignfull is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group alignfull is-style-default" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|70"} -->
+<div style="height:var(--wp--preset--spacing--70)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"verticalAlignment":"center","align":"wide","style":{"spacing":{"blockGap":{"left":"var:preset|spacing|70"}}}} -->
+<div class="wp-block-columns alignwide are-vertically-aligned-center"><!-- wp:column {"verticalAlignment":"center","width":"40%","style":{"spacing":{"padding":{"bottom":"var:preset|spacing|40"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-column is-vertically-aligned-center" style="padding-bottom:var(--wp--preset--spacing--40);flex-basis:40%"><!-- wp:heading -->
+<h2 class="wp-block-heading">Craft and thoughtfullness</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>With high-quality materials and expert craftsmanship, our new collection is built to last and exceed your expectations.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|30","bottom":"var:preset|spacing|30"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--30);padding-bottom:var(--wp--preset--spacing--30)"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left","orientation":"vertical"}} -->
+<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:image {"id":588,"width":"24px","sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="https://kiri.mystagingwebsite.com/wp-content/uploads/2025/02/icon-star-empty.png" alt="A black outlined star icon representing excellence." class="wp-image-588" style="width:24px"/></figure>
```

Comment:
This `src` attribute needs to be updated to `<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/icon-star-empty.png` before merging.